go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/vladimir-buzuev/gohs
+module  github.com/flier/gohs
```

Comment:
Name of the PR says it should be `github.com/blubracket/gohs`?

Reply:
we should use original name. Not sure why I did modify it for the first fork. Updated PR description...